src/pages/ProfileHome.jsx

```diff
@@ -115,69 +117,73 @@ export default function ProfileHome({userEmailParameter, resetUserEmail}) {
     <div className="DependentListItem">
       <Link to="/AddDependent">
         <div className="DependentPhoto">
-          <img src={dependent_add_pic} alt="add dependent"/>
+          <img src={dependent_add_pic} alt="add dependent" />
         </div>
       </Link>
       <div className="DependentName"><p>Add one</p></div>
-      </div>
-    );
-  
+    </div>
+  );
+
   const returnProfilePic = () => {
-      if (userData.name == null || typeof userData.name === 'undefined') {
-          return profile_photo;
-      }
-      if (userData.name == "Shivaji Prafull") {
-        return svichare_photo
-      }
-
-      return profile_photo
+    if (userData.name == null || typeof userData.name === 'undefined') {
+      return profile_photo;
+    }
+    if (userData.name == "Shivaji Prafull") {
+      return svichare_photo
+    }
+
+    if (user.image_url == null || typeof user.image_url === 'undefined') {
+      return profile_photo;
+    }
+
+    return user.image_url;
   }
 
   const returnWelcomeMessage = () => {
-      if (userData.name == null || typeof userData.name === 'undefined') {
-          return "Welcome new user. Update your profile using options below.";
-      }
-
-      if ( userData.name === "Error") {
-          return "Error connecting to cloud. Try refreshing this page.";
-      }
-
-      if ( userData.name === "Mock Value") {
-          return "Loading..";
-      }
-      
-      if (userData.name == "") {
-        return "Welcome new user. Update your profile using options below.";
-      }
-
-      return "Welcome back " + userData.name;
+    if (userData.name == null || typeof userData.name === 'undefined') {
+      return "Welcome new user. Update your profile using options below.";
+    }
+
+    if (userData.name === "Error") {
+      return "Error connecting to cloud. Try refreshing this page.";
+    }
+
+    if (userData.name === "Mock Value") {
+      return "Loading..";
+    }
+
+    if (userData.name == "") {
+      return "Welcome new user. Update your profile using options below.";
+    }
+
+    return "Welcome back " + userData.name;
   }
-  
+
   return (
-      <div className="ProfileHomeContainer">
+    <div className="ProfileHomeContainer">
       {loading ? (
         // Show a loading screen when loading is true
         <div className="LoadingPage"> <h2>Loading...</h2> </div>
-        ) : (
+      ) : (
         <div>
           <div className="ProfileHomeMain">
             <div className="ProfileHomeTopbar">
               <div className="ProfileImage">
-                <img src={returnProfilePic()} alt="profile_photo" />
+                <img src={returnProfilePic()} style={{ width: '100px', height: '100px' }} alt="profile_photo" />
```

Comment:
I had read in some docs about setting actual width and making height auto, to accomodate non-square photos. Does that issue exist for google profile pictures ? or are they always square

Reply:
100 * 100 looks a little small. Btw, you can use the ProfileImage css class for size and also to make the image size  slightly dynamic based on if it is mobile or desktop.

Reply:
Makes sense. Will implement the suggested changes

Reply:
> I had read in some docs about setting actual width and making height auto, to accomodate non-square photos. Does that issue exist for google profile pictures ? or are they always square

Google image thumbnails that are returned by auth are always square as far as I know.